.pre-commit-config.yaml

```diff
@@ -44,12 +44,7 @@ repos:
   hooks:
   - id: pylint
     args:
-    - --jobs=0
     - --rcfile=.pylintrc
```

Comment:
This can probably be deleted too as it's the default config filename.
```suggestion
```

Reply:
Thank you. I deleted this argument also.

Reply:
Marking as resolved.